### PR TITLE
[T6A3][F09-C2] Yuan Wenyi

### PR DESCRIPTION
**Pull request by ywy13250**

### CODE REVIEW

**Comment by ywy13250** [comment]
Work in progress

File: src/seedu/addressbook/storage/Storage.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.storage;
+
+import java.nio.file.Path;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.*;
+
+public abstract class Storage {
+	    /** Default file path used if the user doesn't provide the file name. */
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/storage/Storage.java]: Still using tab??? 
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

File: src/seedu/addressbook/storage/Storage.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.storage;
+
+import java.nio.file.Path;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.*;
+
+public abstract class Storage {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/storage/Storage.java]: This class should have a header comments. If not, developers who needs to inherit from this class will not know how exactly they should override the methods and other developers who write client code for this class will not know how to use it either.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments. If you have more time you can try to implement DI and OCP.

**Pull request closed without merging**